manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~>5.2.4", ">=5.2.4.3"
```

Comment:
Can we apply https://github.com/ManageIQ/activerecord-id_regions/pull/18/files#r537735807 here as well?

Reply:
I don't see why we would, as this isn't a gem that ever needs to be released, and is only used by `ManageIQ/manageiq`, which now requires Rails 6.  The only place where I think this does make sense is in gems that are released to rubygems.org or rubygems.manageiq.org which this is not.

(This is going to be a carbon copy answer that I will be pasting in all provider repos where this comment exists)